src/coreclr/src/pal/src/misc/jitsupport.cpp

```diff
@@ -13,6 +13,134 @@ SET_DEFAULT_DEBUG_CHANNEL(MISC);
 #include <asm/hwcap.h>
 #endif
 
+#if defined(HOST_ARM64) && defined(__linux__)
+typedef struct {
```

Comment:
A style nit: can you please use the C++ style for struct definitions (without the typedef) and also place the opening `{` at a separate line?